Software/bsw/static/Mcal/CAN/src/Can.c

```diff
@@ -477,7 +476,7 @@ void Can_Init( const Can_ConfigType* Config)
                     /* Set cuurent hardware message as the last one in FIFO */
                     HWREG(BaseAddress + CAN_O_IF2MCTL) |= CAN_IF2MCTL_EOB ;
                     /* Map the Current Software HRH with its hardware messages used in the buffer */
-                    MessageObjAssignedToHRH[HRHCount].HRHId = Global_Config->CanHardwareObjectRef[HOHCount].CanObjectId;
+                    MessageObjAssignedToHRH[HRHCount].HRHIndex = HOHCount;
```

Comment:
Same here?

Reply:
AS I said before I need the index .. so from this element in the struct I can get all the information needed about this HOH by writting something like that 

   /*Get HTH index in the CanHardwareObject array */
        index = MessageObjAssignedToHTH[counter].HTHIndex ;
  /*Only check messages that were polling enabled */
        if (TRUE == Global_Config>CanHardwareObjectRef[index].CanHardwareObjectUsesPolling )

Reply:
You assumed that this element will be found all the time.
Up to the SWS:

> This parameter shall exist if CanRxProcessing/CanTxProcessing is set to Mixed